tests/integrationTests/tests/tutorial_16_docker_network_python/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Necessary imports. Provides library functions to ease writing tests.
-from lib import prebuild, testcase, SUBMITTY_TUTORIAL_DIR
+from lib import prebuild, testcase, SUBMITTY_INSTALL_DIR, SUBMITTY_TUTORIAL_DIR
 
 import subprocess
 import os
```

Comment:
Should this test have a schema validation testcase, or is that something you're getting to later?

Reply:
I'll add that one later. PR #4151 will begins to clear a path towards allowing us to have integration tests of docker and docker network grading. When those testcases are added, I'll add the schema validation.